### PR TITLE
Documentation updates

### PR DESCRIPTION
**Pull request by jacksgt**

### CODE REVIEW

File: docs/how-to-use.md [hunk 1]
```diff
@@ -218,19 +218,19 @@ metadata:
...
-         # Note that unlike in pod-single-repo-subpath.yaml, in this
-         # case we don't set mountPropagation nor subPath.
+    - name: demo
+      image: busybox
+      imagePullPolicy: IfNotPresent
+      command: [ "/bin/sh", "-c", "trap : TERM INT; (while true; do sleep 1000; done) & wait" ]
+      volumeMounts:
+        - name: my-cvmfs
+          mountPath: /my-cvmfs
+          # CVMFS automount volumes must be mounted with HostToContainer mount propagation.
```
Comment (gman0) [inline docs/how-to-use.md:228]: If this demo doesn't work as the comment says, then that's a bug in the driver, not an issue with docs. You can keep the indentation fix though!

Comment (gman0) [reply docs/how-to-use.md:228]: I.e. it *should* be possible to mount without `HostToContainer` if the PVC is mounted with `repository` parameter.

Comment (jacksgt) [reply docs/how-to-use.md:228]: Sorry, I copy-pasted the wrong snippet!

Comment (jacksgt) [reply docs/how-to-use.md]: Updated. Could you take another look?

**Review by gman0 — APPROVED** [review]
Thanks @jacksgt!

**Pull request merged**